src/cmake/dependencies.cmake

```diff
@@ -12,7 +12,7 @@ if(NOT USE_SYSTEM_MAGNUM)
   set(WITH_INTERCONNECT OFF CACHE BOOL "" FORCE)
   # Ensure Corrade should be built statically if Magnum is.
   set(BUILD_PLUGINS_STATIC ON CACHE BOOL "" FORCE)
-  set(BUILD_STATIC ON CACHE BOOL "" FORCE)
+  set(BUILD_STATIC OFF CACHE BOOL "" FORCE)
```

Comment:
Yeah, this is what is breaking the rpath dynamic loading. Is this really necessary for the Python viewer, if so we may need to rethink how to handle linking to magnum/corrade plugins a bit.

Reply:
Yes, unfortunately this is necessary until we can fix some symbol duplication issues with the magnum static build. @mosra is aware of the issues and we have some solutions in mind for the longer term. 

For now, we can add some help text to suggest this fix before the merge or leave this PR open until we implement a fix.

Reply:
I should have a fix for the default framebuffer duplication ready by tomorrow, will ping you when that's done. Together with the `RTLD_GLOBAL` flag that needs to be switched for Python module loading (mentioned on Slack) static build should work again.

Reply:
@aclegg3 This will also break pip and possibly conda installs if it's left as a default.